index.js

```diff
@@ -61,8 +61,13 @@ const getNewUsers = async () => {
  * Farcaster API helper functions
  */
 const fetchWithLog = async (url) => {
+  const token = process.env.FARCASTER_BEARER_TOKEN
```

Comment:
nit: can we move this to the top of the file along with other constants?

Reply:
this was removed in @mkochakian 's cleanup!